### ISSUE TITLE
Revoke a mistaken or fraudulent duplicate of a birth or death registration

### ISSUE THREAD

**Issue by euanmillar** [opencrvs/opencrvs-core#4569]
As it is possible for a Registrar to make a duplicate intentionally either because of a mistake or because they have been defrauded ...

And as it is possible for a death registration to be made against an individual who has not actually died, either by mistake or fraudulently ...

There must be a process do revoke. registration.   This is required functionality for MOSIP integration at ID4Africa

**Comment by rikukissa** [comment]
Questions: 

- How does Revoking a record affect the records behaviour and processes around it? Is it still visible in workqueues? Can it be searched for? Are there any UI implications?
- Is the templated approach where we only allow capturing the "metadata" enough? There are some country requirements today for hiding or changing "Who's collection?" options or the options in who is making a correction e.g. https://github.com/opencrvs/opencrvs-core/issues/7057

**Comment by jpye-finch** [comment]
Adds ACs for your first bullet. Will be adding more as this is flecthed out

**Comment by prinzab** [comment]
Uganda Requirement

**Revoke a registration**

As a user, I want the ability to cancel (revoke) a registration due to errors or suspected fraudulent information, so that inaccurate records can be removed and flagged as invalid. 

Acceptance criteria
- Executive Director, and by invitation, the Legal Department are involved in investigations when revocation is triggered by a complaint
- Registration Officer can trigger revocation for records with issues, such as suspected duplicates
- Once a certificate is revoked, its QR code shows invalid status when scanned
- System sends notification to the certificate collector
- Revoked records display an "Invalid" status in the system
- Requires: Office Hierarchy and New System User Type (Senior Registrar)
